### PR TITLE
Verification response with `verified` and `verifications`

### PR DESCRIPTION
**Pull request by nissimsan**
Ref https://github.com/w3c-ccg/traceability-interop/issues/176 and Tuesday's discussion.

### CODE REVIEW

**Comment by nissimsan** [comment]
Hmm - not clear what makes the check fail?

File: docs/openapi/schemas/VerificationResult.yml [hunk 1]
```diff
@@ -3,15 +3,23 @@ type: object
...
-  warnings:
-    type: array
-    items:
-      type: string
-  errors:
-    type: array
-    items:
-      type: string
+      $ref: ./Verification.yml
+example:
```
Comment (BenjaminMoe) [inline docs/openapi/schemas/VerificationResult.yml:10]: Should `verified` and `verifications` be required?

Comment (OR13) [reply docs/openapi/schemas/VerificationResult.yml:10]: imo verified yes, verifications no.

**Review by OR13 — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

**Comment by nissimsan** [comment]
@mkhraisha, FYI - would you want to bring this upstream to the VC-API? Cheers!

**Comment by mkhraisha** [comment]
Yes, I assigned myself to the issue, I'll take it upstream

**Review by mprorock — APPROVED** [review]

**Pull request merged**